src/main/kotlin/com/terraformation/backend/accelerator/api/CohortsController.kt

```diff
@@ -100,6 +106,34 @@ class CohortsController(
     return getCohort(cohortId)
   }
 
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{cohortId}/modules")
+  @Operation(summary = "List modules.")
+  fun listModules(
```

Comment:
A minor thing: rename this to something like `listCohortModules` and the other method to `getCohortModule`. By default, the OpenAPI schema generator uses the controller method name as the name of the operation, and operation names have to be unique in a schema. So it ends up appending suffixes like `_1` to the duplicate operation names to make them unique, which is fine initially, but then if we ever get rid of or rename the existing `listModules` method, it has the side effect of renaming this endpoint's operation (because the suffix is no longer needed).

If you look at the output of the ["Diff OpenAPI docs"](https://github.com/terraware/terraware-server/actions/runs/11923050776/job/33230594392#step:14:1) step of the CI build, you'll see what I'm talking about.